src/profile-logic/marker-schema.js

```diff
@@ -381,7 +393,8 @@ export function getLabelGetter(
 export function formatFromMarkerSchema(
   markerType: string,
   format: MarkerFormatType,
-  value: any
+  value: any,
+  stringTable: ?UniqueStringArray
```

Comment:
Is there a reason why you wanted to make this `stringTable` parameter optional? I think it might make it more error-prone in the future when we need to use it (seeing it being only optional and not passing it at all, thinking that it's not needed even though it is). Can you make it non-optional if there isn't an issue that I don't see?

Also probably you'll need to update the existing tests, I think you can just pass `new UniqueStringArray()` to most of them to fix the current ones. But of course it could be great to have a test for this new type as well.

Reply:
From what I recall, it's due to this function being used in multiple contexts, including some where we might not have a stringtable easily available (without further refactoring). I may be wrong there, and making it mandatory should answer that question.

There are already some updates to tests that use the default thread mechanism to get an empty string table, but I agree that it needs more tests.

Reply:
So, looking into this again, I can confirm that there are other places in the frontend, such as `_renderMoreInfoSection` in `MetaInfo.js`, where we call this function *without* (as far as I'm aware) an available stringTable. I think it's something we could look into changing in future, but I'm hesitant to include it in this PR as getting access to the stringTable in these contexts is not something I'm confident in implementing.